install-application/action.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'install-application'
+description: 'Power Platform Install Application'
```

Comment:
nit: append "... from AppSource"
just to make the context clearer

Also: please add a "(Preview): Power P...." prefix to this description, since it is indeed public preview only from the AppMgmt service side